vuepress/docs/next/tutorials/create/mfe/angular.md

```diff
@@ -74,245 +88,172 @@ This is the expected output:
     │   ├── styles.css
     │   └── test.ts
     │
+    ├── .browserlistrc
     ├── .editorconfig
     ├── .gitignore
     ├── angular.json
-    ├── browserlist
     ├── karma.conf.js
     ├── package.json
+    ├── package-lock.json
     ├── README.md
     ├── tsconfig.app.json
     ├── tsconfig.json
-    ├── tsconfig.spec.json
-    └── tslint.json
+    └── tsconfig.spec.json
+    
 
-### Convert to Custom Element
-
-Next, let's convert our Angular app into a custom element. We'll use [Angular elements](https://angular.io/guide/elements) to transform components into custom elements.
+### Configure the Custom Element
 
+Next, convert the Angular widget into a custom element, using [Angular elements](https://angular.io/guide/elements) to transform components into custom elements.
+1. Install the Angular elements
 ``` bash
 ng add @angular/elements
 ```
+[Angular elements](https://angular.io/guide/elements) are Angular components packaged as custom elements (also called Web Components), a web standard for defining new HTML elements in a framework-agnostic way.
```

Comment:
"a framework-agnostic web standard"

Reply:
strictly speaking, custom elements and two other related technologies comprise web components; custom elements is not synonymous with web components

Reply:
Agreed, custom elements is part of Web Components, but not the same thing. https://developer.mozilla.org/en-US/docs/Web/Web_Components